### ISSUE TITLE
Debugging removes project directory (when manually using wrong debugging configuration type "V8 Debugger for node")

### ISSUE THREAD

**Issue by mattispasch** [Nodeclipse/nodeclipse-1#104]
Hello, 

if i have a project in eclipse, lets say "my-server" and I create a debugging configuration with the "V8 Debugger for node" named "my-server", running the debugger deletes my project directory.

I think this bug actually is really critical as I lost some hours work because of it.

Thanks, Mattis

Nodeclipse: 0.8.0
Eclipse 4.3.0 (Juno)

**Comment by Redsandro** [comment]
OOOOH my ffffffff....

I just lost hours of work with this exact issue in Eclipse Mars 4.5.0.
Any idea if this temporary directory is still anywhere to be found?

**Comment by paulvi** [comment]
@Redsandro Was your debug configuration name exactly as project name?

**Comment by Redsandro** [comment]
Yes indeed. Not the same as the actual project directory, but the same as the project name.

The `V8 Debugger for Node` is more useful than `Node Application` because the former allows script mapping so that you can break in your own source files. The latter breaks on chromium temporary files so you need to swap to a different editor to edit your code.
